src/features/import-maps.js

```diff
@@ -24,15 +24,24 @@ if (hasDocument) {
   window.addEventListener('DOMContentLoaded', processScripts);
 }
 
+export var inlineScripts = {}
```

Comment:
Please add semicolons to all code in this pull request.

```suggestion
export var inlineScripts = {};
```